go/felix/usagerep/usagerep.go

```diff
@@ -54,6 +50,19 @@ func PeriodicallyReportUsage(interval time.Duration, hostname, clusterGUID, clus
 	}
 }
 
+func calculateInitialDelay(numHosts int) time.Duration {
+	// To avoid thundering herd, inject some startup jitter.
+	if numHosts <= 0 {
+		// Intn requires a positive argument so clamp numHosts to be >= 1.
+		numHosts = 1
+	}
+	initialJitter := time.Duration(rand.Intn(numHosts*1000)) * time.Millisecond
```

Comment:
Also I'm wondering if a paren is misplaced here, and it should be:

    initialJitter := time.Duration(rand.Intn(numHosts*1000) * time.Millisecond)

(Because it appears that the other `time.` methods around here don't handle plain integers.)